test/parallel/test-zlib-from-concatenated-gzip.js

```diff
@@ -16,3 +18,23 @@ zlib.gunzip(data, common.mustCall((err, result) => {
   assert.ifError(err);
   assert.equal(result, 'abcdef', 'result should match original string');
 }));
+
+// files that have the "right" magic bytes for starting a new gzip member
+// in the middle of themselves, even if they are part of a single
+// regularly compressed member
+const pmmFile = path.join(common.fixturesDir + '/pseudo-multimember-gzip');
```

Comment:
If you're using `path.join` here it should be `path.join(common.fixturesDir, 'pseudo-multimember-gzip');` rather than concat the strings.